app/src/main/java/com/github/se/signify/ui/screens/challenge/ChallengeScreen.kt

```diff
@@ -292,14 +298,17 @@ fun OngoingChallengeCard(
                 }
             val opponentText = stringResource(R.string.opponent_text)
             Text(
-                text = "$opponentText $opponentName",
+                text = "$opponentText: $opponentName",
                 fontSize = 18.sp,
-                color = MaterialTheme.colorScheme.onSurface,
-                fontWeight = FontWeight.Bold)
+                color = MaterialTheme.colorScheme.onSurface)
             Text(
                 text = "${stringResource(R.string.mode_text)}: ${challenge.mode}",
                 fontSize = 14.sp,
                 color = MaterialTheme.colorScheme.onSurface)
+            Text(
+                text = "Round: $round/3",
```

Comment:
Also here Round should not be hardcoded even if in French

Reply:
Done, ty